go/teams/chain.go

```diff
@@ -519,11 +519,22 @@ func (t *TeamSigChainState) FindActiveInviteByID(id keybase1.TeamInviteID) (keyb
 	return invite, found
 }
 
-func (t TeamSigChainState) IsInviteObsolete(id keybase1.TeamInviteID) bool {
+func (t *TeamSigChainState) IsInviteObsolete(id keybase1.TeamInviteID) bool {
 	_, ok := t.inner.ObsoleteInvites[id]
 	return ok
 }
 
+func (t *TeamSigChainState) FindActiveKeybaseInvite(uid keybase1.UID) (keybase1.TeamInvite, keybase1.UserVersion, bool) {
```

Comment:
There could be more than one (god help us) so this returns the first. Right?

Reply:
Heh my assumption was that at least for implicit team there can only be one existing membership or invite. This was later proven to be not true (and probably there are teams like that in the wild), but I think we can still get away with reAddMember only sweeping one invite, at least for now. Or it could just use transaction.go code and sweep everything. What do you think?